src/Tizen.NUI/src/internal/Interop/Interop.GraphicsCapabilities.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class GraphicsCapabilities
+        {
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_IsBlendEquationSupported")]
+            public static extern bool IsBlendEquationSupported(int blendEquation);
+        }
+    }
+}
```

Comment:
please add new line at the end of file from next time~
( https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file )